dictionary/lib/dictionary/agent.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Dictionary.Agent do
+  alias Dictionary.WordList, as: WordList
+
+  @me DICTIONARYagent
+
+  def start_link() do
+    Agent.start_link(fn -> WordList.word_list end, name: @me)
+  end                                                           # End of start_link
```

Comment:
No deduction, but I don't this the `# end of ...` comments are needed for single line functions :)